src/pages/scientificServices/pipelines/views/JobHistory.tsx

```diff
@@ -124,7 +137,11 @@ export const JobHistory = () => {
   );
 };
 
-const getColumns = (paginatedRuns: PipelineRun[]) => {
+const getColumns = (
+  paginatedRuns: PipelineRun[],
+  sort: { field: string; direction: 'asc' | 'desc' },
```

Comment:
```
field: string; direction: 'asc' | 'desc'
```
is used in a couple of places, is it possible to refactor that or is this kind of how it has to be?

Reply:
per discussion above, cleaned up in 6e786599d59b6806899d1a3d6c429a9989319e83